frontend/src/components/filters/Sidebar.svelte

```diff
@@ -9,23 +9,23 @@
   import VvLogo from "./VvLogo.svelte";
 </script>
 
-<div class="sticky top-0 h-screen">
-  <div class="wrapper">
+<div class="wrapper sticky h-screen top-0">
```

Comment:
```suggestion
<div class="wrapper sticky top-0">
```

<img width="392" alt="Screenshot 2022-10-19 at 11 10 35" src="https://user-images.githubusercontent.com/69513935/196648795-649342f6-a4e9-4bb9-be95-ad6433f1bc47.png">
it looks like this when I scroll down 😅 Removing h-screen works.

For later I think we maybe should make it fixed? so it wont be scrollable.

Reply:
lol also sorry for being naggy lol, lgtm after this hahahah